src/panel/poi/PoiHeader.jsx

```diff
@@ -8,15 +9,30 @@ const PoiHeader = ({ poi }) => {
   const title = name || localName || poiSubClass(subClassName);
 
   return <div>
-    <h4 className="poi_panel__title">
-      <span className="poi_panel__title__main">{title}</span>
-      {name && localName && localName !== name &&
-        <p className="poi_panel__title__alternative">{localName}</p>
+    {subClassName === 'latlon' && <div>
+      {address && address.label &&
+        <div className="poi_panel__pre_title">{ _('Close to', 'poi')}</div>
       }
-    </h4>
-    {subClassName && <p className="poi_panel__description">{poiSubClass(subClassName)}</p>}
-    {address && address.label && <p className="poi_panel__address">{address.label}</p>}
-    {grades && <ReviewScore reviews={grades} poi={poi} />}
+      <h4 className="poi_panel__title">
+        <span className="poi_panel__title__main">
+          {address && address.label ? address.label : title}
+        </span>
+      </h4>
+      {<p className="poi_panel__address">{address && address.label ? title : ''}</p>}
```

Comment:
Do we really need this node when it's empty? The previous impl. didn't add it at all (but maybe it's needed now for layout reasons).